tools/ci/update-vulnerable-dependencies/update-vulnerable-dependencies.sh

```diff
@@ -5,14 +5,15 @@ set -e
 command -v osv-scanner >/dev/null 2>&1 || { echo >&2 "osv-scanner not installed!"; exit 1; }
 command -v jq >/dev/null 2>&1 || { echo >&2 "jq not installed!"; exit 1; }
 
+KUMA_DIR=${1:-"."}
 
 for dep in $(osv-scanner --lockfile=go.mod --json | jq -c '.results[].packages[] | .package.name as $vulnerablePackage | {
   name: $vulnerablePackage,
   current: .package.version,
   fixedVersions: [.vulnerabilities[].affected[] | select(.package.name == $vulnerablePackage) | .ranges[].events |
   map(select(.fixed != null) | .fixed)] | map(select(length > 0)) } | select(.name != "github.com/kumahq/kuma")'); do
 
-  fixVersion=$(go run tools/ci/update-vulnerable-dependencies/main.go <<< "$dep")
+  fixVersion=$(go run "$KUMA_DIR"/tools/ci/update-vulnerable-dependencies/main.go <<< "$dep")
```

Comment:
Isn't just the same dir as the one in the script? Really feel like this should be moved to `kumahq/ci-tools`

Reply:
```suggestion
  dir=$( dirname -- "${BASH_SOURCE[0]}" )
  fixVersion=$(go run "$dir"/main.go <<< "$dep")
```

Reply:
You are right with the same dir. I'm gonna update the PR. I would prefer to first fix the current solution, and in a following up PR to potentially move it to `kumahq/ci-tools`